test/formatters/installers/dprint.bash

```diff
@@ -0,0 +1,11 @@
+npm install -g dprint
+cat <<\EOF >/tmp/dprint.json
+{
+  "toml": {},
+  "excludes": [],
+  "plugins": [
+    "https://plugins.dprint.dev/toml-0.5.4.wasm"
+  ]
+}
+EOF
+dprint config update
```

Comment:
This feels dirty, but gets things going. As far as the tests are run in a subdir of /tmp somewhere -- dprint searches config from current dir and ancestors.

We could add a bunch of other plugins and test files if wanted. Can add if so, once the `file-name-nondirectory` issue above is solved.

Reply:
Yeah, multiple `in.*` / `out.*` pairs is supposed to work. See the `prettier` subdir for example.

Agree that this doesn't look great, but I think the real answer is there needs to be a more mature test framework that would allow the inclusion of extra files to be placed on the filesystem during test invocation for a particular formatter, but it is okay to not bother with that for now since this works.

Reply:
Ok, added some more plugins and sample code in e13e245dcce7acd8561f2cf28a32db1a9419d43b.